plugins/helm/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+find_package(yaml-cpp REQUIRED)
```

Comment:
This should be moved to the `CMakeLists.txt` file of the parser component of the plugin, as it is only used there.